### PR TITLE
add google books api util

### PR DESCRIPTION
**Pull request by festinalente91**
- add google books api util class.

### CODE REVIEW

File: app/books.util.js [hunk 1]
```diff
@@ -34,23 +48,42 @@ export default class BookAPIUtil {
...
-    const address = `${GOOGLE_BOOKS_API.URL}?q=${keyword}&key=${GOOGLE_BOOKS_API.API_KEY}`;
+  static searchBooks(keyword, { orderBy = 'relevance', maxResults = 5, startIndex = 0, printType = 'books' }) {
+    const options = new Map()
+      .set('orderBy', orderBy)
+      .set('maxResults', maxResults)
+      .set('startIndex', startIndex)
+      .set('printType', printType);
+
+    let query = '';
+    options.forEach((value, key) => {
```
Comment (ochanje210) [inline app/books.util.js]: using for-of loop is preferred. here are some ref: http://stackoverflow.com/questions/22155280/why-is-native-javascript-array-foreach-method-significantly-slower-than-the-stan
and we can use it like below:
```jsx
for (let [key, value] of options) {
  query += `&${key}=${value}`;
}
```

Comment (ochanje210) [reply app/books.util.js]: also, here is the difference between for-in loop and for-of loop: https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/

Comment (ochanje210) [reply app/books.util.js]: feel free to correct me

File: app/books.util.js [hunk 2]
```diff
@@ -80,9 +80,9 @@ export default class BookAPIUtil {
       .set('printType', printType);
 
     let query = '';
-    options.forEach((value, key) => {
+    for (let [key, value] of options) {
       query = query + `&${key}=${value}`;
-    });
+    }
```
Comment (ochanje210) [inline app/books.util.js]: semicolon

**Comment by ochanje210** [comment]
LGTM

**Comment by ace0625** [comment]
lgtm

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-client/pull-request/16/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-client)

**Pull request merged**